proto/axelarcork/v1/proposal.proto

```diff
@@ -32,13 +30,13 @@ message RemoveAxelarManagedCellarIDsProposal {
   CellarIDSet cellar_ids  = 4;
```

Comment:
Was this meant to be changed to `repeated string` as well like the WithDeposit version?

Reply:
No, this is mirroring the way the `cork` proposals work, which is also a bit janky, but it simplifies the CLI format, especially when you add in chain IDs.